lib/assessment.js

```diff
@@ -8,15 +8,21 @@
 // Question 1
 // Write a callback for reduce that selects
 // the minimum value in the array
-var min = function min() {
-
+var min = function min(array) {
+  var minimum = array.reduce(function(a,b){
+    return b < a ? b : a;
+  });
+  return minimum;
```

Comment:
This is not the correct solution, in that it can't be passed in as a callback to `Array.reduce`, but it does *contain* the correct solution.

Assign the following to `min` and you're golden.
```js
function(a,b){
    return b < a ? b : a;
}
```

Full credit for containing the correct answer as an anonymous function.